user_home/templates/customer_homepage.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+
+{% block menu %}
+<li class="list-group-item list-group-item-light menu" id="makeAppointment">
+  <a href="http://{{ request.get_host }}/user_home/appointment"> Make Appointment </a>
+</li>
+<li class="list-group-item list-group-item-light menu" id="updateProfile">
+  <a href="http://{{ request.get_host }}/user_home/profile_update/"> Update Profile</a>
+</li>
```

Comment:
what about transactions here?

Reply:
I am not handling anything realted to customer